src/services/ledger/domain/index.types.d.ts

```diff
@@ -3,6 +3,11 @@ type LedgerAccountId = string & { [ledgerAccountId]: never }
 
 type TxMetadata = any
```

Comment:
as far as I remember we did not have this alert. could you please add an eslint exception?